deploy/olm-catalog/mattermost-operator/0.5.0/mattermost-operator.v0.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,269 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    categories: "Collaboration Tool"
+    alm-examples: |-
+      [
+        {
+          "apiVersion":"mattermost.com/v1alpha1",
+          "kind":"ClusterInstallation","metadata": {
+            "name":"example-clusterinstallation"
+          },
+          "spec": {
+            "ingressName":"example.mattermost-example.dev"
+          }
+        }
+      ]
+    capabilities: Seamless Upgrades
+    certified: "false"
+    support: Carlos Panato
+    containerImage: docker.io/mattermost/mattermost-operator:v0.5.0
+    repository: https://github.com/mattermost/mattermost-operator
+    description: Mattermost is a flexible, open source messaging platform that enables secure team collaboration.
+    createdAt: 2019-07-18T09:42:00Z
+  name: mattermost-operator.v0.5.0
+  namespace: placeholder
+spec:
+  version: 0.5.0
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: ClusterInstallation
+      name: clusterinstallations.mattermost.com
+      version: v1alpha1
+  description: |-
+    [Mattermost](https://mattermost.com) is a flexible, open source messaging platform that enables secure team collaboration.
```

Comment:
@jwilander was reviewing the guideline to add the operator to the hub and we need to add some description, could you please take a look in this section and add more things if we need to? thanks!

Reply:
This is good for now 👍